fili-core/src/main/java/com/yahoo/bard/webservice/web/apirequest/DataApiRequestImpl.java

```diff
@@ -1316,6 +1317,7 @@ protected void validateTopN(String topNRequest,  int topN, LinkedHashSet<OrderBy
                             pathSegment -> dimensionDictionary.findByApiName(pathSegment.getPath()),
                             pathSegment -> bindShowClause(pathSegment, dimensionDictionary),
                             (LinkedHashSet<DimensionField> e, LinkedHashSet<DimensionField> i) -> {
+                                // I do not think you should  be modifying the provided LinkedHashSet here
                                 e.addAll(i);
```

Comment:
@michael-mclawhorn I suspect this is the error.  We should not be modifying the getDefaultDimensionFields  set here.